### ISSUE TITLE
Actually delete conversations

### ISSUE THREAD

**Issue by someoneEsle** [wireapp/wire-android#548]
Currently, when I delete a conversation two things happen:
- the conversation is archived;
- its content disappears, *but is not actually deleted from the phone*.

Besides the fact that when I tap "delete" I actually mean "delete" (and not "hide from me but stay on the phone"), I reeeaally need a way to limit how much memory Wire is using, something that isn't "clearing the app's data" altogether. Some conversations take up a lot of space, a great space, the greatest space you've ever seen. The space they consume is yuge; it's so big, it's unbelievable. People call me to tell me how much space they consume, and I say to them, we're gonna fix it, we're gonna take those conversations and send them out of the memory, it's gonna happen, and it's gonna be amazing. Believe me.

**TL;DR:** Deleting a conversation should actually delete its content from the app's storage instead of just hiding it from the user.
(This is related to #326, and I'm surprised no one opened an issue about this yet)

**Comment by someoneEsle** [comment]
Jeeeeez I just found #318

**Comment by Zahlentheorie** [comment]
Will this be changed? Some friends of mine left Wire because they haven't got enough space on their phones, and Wire produced about 1 GB of data.

**Comment by someoneEsle** [comment]
I fear I'm going to face the same problem with my friends soon. I myself have to reinstall wire every once in a while, which doesn't make me too happy

**Comment by Zahlentheorie** [comment]
Everytime I am reinstalling Wire my friends have to authorize my new phone etc. It's not a satisfying solution, and I don't feel like they will change this problem in the next few months.

**Comment by makingthematrix** [comment]
Fixed in https://github.com/wireapp/wire-android/pull/2389
